src/m365/spo/commands/user/user-get.ts

```diff
@@ -119,6 +186,26 @@ class SpoUserGetCommand extends SpoCommand {
       this.handleRejectedODataJsonPromise(err);
     }
   }
+
+  private async getUser(options: GlobalOptions): Promise<any> {
+    const requestUrl: string = `${options.webUrl}/_api/web/siteusers?$filter=UserPrincipalName eq ('${formatting.encodeQueryParameter(options.userName)}')`;
+    const requestOptions: CliRequestOptions = {
+      url: requestUrl,
+      headers: {
+        accept: 'application/json;odata=nometadata'
+      },
+      responseType: 'json'
+    };
+
+    const userInstance = await request.get(requestOptions);
```

Comment:
Hi @MathijsVerbeeck 

We usually use `odata.getAllItems` only in `list` commands. This being a `get` command, should be more appropriate to use `request.get`?

Please suggest.

Reply:
Hi @SmitaNachan 

Great remark. The reasoning on why using odata.getAllItems instead of request.get for me was that the output from the request.get is the exact same as the odata.getAllItems, and the odata.getAllItems will immediately return the array. But maybe in this case you are right. Let's keep it as you implemented it.